todo_app/templates/item.html

```diff
@@ -2,18 +2,18 @@
   <div class="input-group-text">
       <form method="post" action="{{ url_for('complete') }}">
           <input class="form-check-input" type="checkbox" onchange="this.form.submit()" {% if status == 'completed' %} checked="checked" {% endif %}>
-          <input type="hidden" name="status" value="{{ status }}">
-          <input type="hidden" name="item_id" value="{{ item.id }}">
+          <input type="hidden" name="status" value="{{ item.status }}">
+          <input type="hidden" name="item_id" value="{{ item._id }}">
```

Comment:
The fact you've had to change this is a good prompt for why we want to use the `Item` class; a change in our underlying data persistence shouldn't change the representation of items in our frontend!